utils/kvrocks2redis/tests/check_consistency.py

```diff
@@ -100,7 +100,7 @@ def _import_and_compare(self, num):
             self.src_cli.incr(incr_key)
             hash_key = f'hash_key_{i}'
             hash_value = {'field1': f'field1_value_{i}', 'field2': f'field2_value_{i}'}
-            self.src_cli.hmset(hash_key, hash_value)
+            self.src_cli.hset(hash_key, hash_value)
```

Comment:
@aleksraiden Seems redis-py didn't support the object as its value.

Reply:
Hmmm, yes, You are right, my mistake, need to rewrite code. Thanks!